public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -9,34 +10,35 @@ include ../_util-fns
 
   In this chapter we'll learn what DI is and why we want it.
   Then we'll learn [how to use it](#angular-di) in an Angular app.
-// #enddocregion intro
-:marked
-  [Run the live example](/resources/live-examples/dependency-injection/ts/plnkr.html)
-// #docregion why-1
-<a id="why-di"></a>
-.l-main-section
+
+  - [Why dependency injection?](#why-dependency-injection)
+  - [Angular dependency injection](#angular-dependency-injection)
+  - [Injector providers](#injector-providers)
+  - [Dependency injection tokens](#dependency-injection-tokens)
+  - [Summary](#summary)
+
+p Run the #[+liveExampleLink2()].
+
+.l-main-section#why-di
 :marked
   ## Why dependency injection?
 
   Let's start with the following code.
-// #enddocregion why-1
+
 +makeExample('dependency-injection/ts/app/car/car-no-di.ts', 'car', 'app/car/car.ts (without DI)')
-// #docregion why-2
-- var lang = current.path[1]
-- var prefix = lang == 'dart' ? '' : 'this.'
+
 :marked
   Our `Car` creates everything it needs inside its constructor.
   What's the problem?
-
   The problem is that our `Car` class is brittle, inflexible, and hard to test.
 
   Our `Car` needs an engine and tires. Instead of asking for them,
-  the `Car` constructor creates its own copies by "new-ing" them from
+  the `Car` constructor instantiates its own copies from
   the very specific classes, `Engine` and `Tires`.
```

Comment:
Remove the "," after "classes" while you're in here. (It's a restrictive phrase, not a parenthetical one.) _Or_ delete the "the" before "very" and keep the comma; that way the phrase becomes parenthetical.